scrnaseq.Rmd

```diff
@@ -1761,8 +1768,56 @@ if (markers_found) {
 }
 ```
 
-### Heatmaps of differentially expressed genes {.tabset}
-#### All up- and down-regulated genes
+## Dotplots per cluster per sample 
+If the dataset contains multiple samples, we can visualise the expression of a gene that is up-regulated in a cluster separately for each sample. For each cluster, we extract up-regulated genes, and visualise expression of these genes in all cells in that cluster, split by their sample of origin. 
```

Comment:
What will happen with the code if there is only one sample?

Reply:
Will give it a try to be sure.

Reply:
It runs through, and the dot plot contains one row, non-scaled as there is only one sample. 

Once we know where we will move this section to, we could move the Headers inside the code chunk and only include the plots if there is more than one sample.

Reply:
In this context, I also fixed the issue that came up with Torsten.